.github/workflows/scan-secrets.yml

```diff
@@ -0,0 +1,44 @@
+name: Perform secret scans
+on:
+  workflow_call:
+    inputs:
+      base:
+        required: false
+        type: string
+        default: ""
+        description: An optional branch to base the scan on.
+      enable_trufflehog_action:
+        required: false
+        type: boolean
+        default: true
+        description: An option to enable the TruffleHog secret scan action.
+      env_vars:
+        required: false
+        type: string
+        default: "{}"
+        description: Extra variables to be passed to the environment.
+      extra_args:
+        required: true
```

Comment:
This shouldn't be required.  If you set required it will mean we have to input it on every calling workflow.